src/cpx_solve.jl

```diff
@@ -195,6 +210,21 @@ function c_api_getax(model::Model, Ax::FVec)
     end
 end
 
+function c_api_getxqxax(model::Model, xqxax::FVec)
```

Comment:
Maybe call `xqxax::FVec` something clearer like `dest::Vector{Float64}`.

Reply:
Done.
I do the same with their linear counterparts? (e.g, `get_ax`)

Reply:
No, only touch the code that you are adding for now.